docs/how-to/h-provision-certificates.md

```diff
@@ -10,13 +10,13 @@ To do so, the TLS Certificates Operator provides two actions:
 The following sections explain how to run both of them.
```

Comment:
AFAIK those docs aren't synced to charmhub right now so I'd suggest simply deleting the `docs` directory from here and modifying the ones on discourse. 

If we don't do that we will very soon have a sync issue where what's here does not match what's on discourse. 

FYI @dariofaccin

Reply:
What do you think @dariofaccin ?

Reply:
I don't see any issues with removing the `docs` folder from here, until the sync action will be available. It also won't be a problem to cherry pick the folder in future, when we'll implement the sync.